modules/ROOT/pages/apikit-odatav2-extension-reference.adoc

```diff
@@ -0,0 +1,41 @@
+= APIkit for OData v2 Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+This reference contains dependency, namespace, and schema information.
```

Comment:
Per our style, we are not supposed to use this type of self-referential information. Can you think of a different lead-in?